README.md

```diff
@@ -82,6 +82,7 @@ software, and may be redistributed under the terms specified in the
 [LICENSE]: https://github.com/thoughtbot/factory_bot/blob/main/LICENSE
 
 <!-- START /templates/footer.md -->
+
```

Comment:
With this change, the workflow will generate an automatic PR with the generated content.